core/lib/tracehouse/trace-processor.js

```diff
@@ -705,7 +705,7 @@ class TraceProcessor {
         return Boolean(
           evt.name === 'FrameCommittedInBrowser' &&
           evt.args.data?.frame &&
-          evt.args.data.url
```

Comment:
https://bugs.chromium.org/p/chromium/issues/detail?id=1453289

This is unrelated, but we need to fix both issues at the same time to land.